k9mail/src/main/java/com/fsck/k9/preferences/FolderSettings.java

```diff
@@ -34,7 +34,7 @@
         s.put("notifyMode", Settings.versions(
                 new V(34, new EnumSetting<>(FolderClass.class, FolderClass.INHERITED))
         ));
-        s.put("syncMode", Settings.versions(
+        s.put("pollMode", Settings.versions(
```

Comment:
The version information is there to be able to import settings exported with older app versions. To not lose the poll/sync mode setting when importing old settings files we have to do the following:
* Add a new entry 'pollMode' with version 48
* Add `new V(48, null)` to 'syncMode' to indicate the setting with that name has been removed
* Add migration code to copy the value from 'syncMode' to 'pollMode'